quinn-proto/src/endpoint.rs

```diff
@@ -351,7 +351,7 @@ where
         remote: SocketAddr,
         server_name: &str,
     ) -> Result<(ConnectionHandle, Connection<S>), ConnectError> {
-        config.transport.validate()?;
+        config.transport_config().validate()?;
```

Comment:
We decided we didn't want getters, right?

Reply:
Because it was a field wrapped in `Arc` I thought it could be useful to add a getter. This to reduce redundant cloning trough out the lib.

Reply:
This has actually added additional unnecessary cloning. It's best to keep these things explicit.